test/integration-gyp-test.js

```diff
@@ -4,27 +4,37 @@ var path = require('path')
 var fs = require('fs')
 var rm = require('rimraf')
 
-var cwd = path.join(__dirname, 'native-module')
+var cwd = path.join(__dirname, 'native-module-gyp')
 
-test('can prebuild a native module for electron', function (t) {
+test('can prebuild a gyp native module for node', function (t) {
   rm.sync(path.join(cwd, 'prebuilds'))
-  var file = 'native-v1.0.0-electron-v50-' + process.platform + '-' + process.arch + '.tar.gz'
+  var file = 'native-v1.0.0-node-v57-' + process.platform + '-' + process.arch + '.tar.gz'
   var prebuild = path.join(cwd, 'prebuilds', file)
   // A quick, temporary fix for a node.js bug (https://github.com/prebuild/prebuild/pull/208#issuecomment-361108755)
-  console.log()
   exec('npm run prebuild', { cwd: cwd }, function (error, stdout, stderr) {
```

Comment:
These `console.log()`s are weird fixes for the nodejs issue in the link above. As you can see node-v4-x86 is failing tests without these. nodejs fixed the issue only for 8.x+

Reply:
Oh sorry I thought I left that in accidentally, I'll add it back